resources/searches/Traffic-Direction-Table.json

```diff
@@ -0,0 +1,20 @@
+{
+   "sort": [
+      "TimeUpdated", 
+      "desc"
+   ], 
+   "hits": 0, 
+   "description": "", 
+   "title": "Traffic Direction Table", 
+   "version": 1, 
+   "kibanaSavedObjectMeta": {
+      "searchSourceJSON": "{\"index\":\"[network_]YYYY_MM_DD\",\"highlight\":{\"pre_tags\":[\"@kibana-highlighted-field@\"],\"post_tags\":[\"@/kibana-highlighted-field@\"],\"fields\":{\"*\":{}},\"fragment_size\":2147483647},\"filter\":[],\"query\":{\"query_string\":{\"query\":\"*\",\"analyze_wildcard\":true}}}"
+   }, 
+   "columns": [
+      "TrafficDirection_NM", 
```

Comment:
TrafficDirection_NM is a custom metadata field.   We can only add this if the the user will get the the kibana cache refreshed somehow.  

Did this get resolved why I was done?  Is the refresh button available for the user or did you solve it in some other way?